### PR TITLE
fix(keeper): mirror runtime trust attention in status

### PR DESCRIPTION
**Pull request by jeong-sik**
## Summary

Keeper status surfaces now mirror embedded `runtime_trust.needs_attention` into the top-level `needs_attention` fields used by `masc_keeper_status` and dashboard keeper JSON.

The new bridge helper preserves stronger local reasons such as pending approvals, but promotes runtime-trust attention when the older local attention path would otherwise say the keeper is healthy.

## Product impact

Operators should no longer see a keeper reported as top-level healthy while the embedded runtime-trust block says the keeper needs attention because the latest execution receipt or terminal reason paused the runtime. This makes silent-action failures and required-tool contract pauses visible in the first status fields that humans and automation scan.

## Evidence

- Root cause: `Keeper_runtime_trust_snapshot` already derives attention from Pause/Alert disposition, but `keeper_status_detail` and dashboard keeper JSON emitted separate top-level attention fields from `attention_fields_json`.
- `attention_fields_with_runtime_trust` now promotes trust attention without overriding existing approval/runtime-blocker attention reasons.
- `keeper_status_detail` and `dashboard_http_keeper` both apply the overlay after building `runtime_trust`.

## Review evidence

- `git diff --check origin/main...HEAD`
- `env DUNE_CACHE=disabled scripts/dune-local.sh build test/test_keeper_exec_status.exe`
- `./_build/default/test/test_keeper_exec_status.exe --color=never` passed, 32 tests.
- `env DUNE_CACHE=disabled scripts/dune-local.sh build test/test_keeper_pause_silent_failure_source.exe`
- `./_build/default/test/test_keeper_pause_silent_failure_source.exe --color=never` passed, 7 tests.
- `env MASC_INCLUDE_OPERATOR_CONTROL=true DUNE_CACHE=disabled scripts/dune-local.sh build test/test_operator_control.exe`
- `./_build/default/test/test_operator_control.exe test operator 8,53 --color=never` passed, 2 targeted tests.

## Linked issue

None.

### CODE REVIEW

**Review by jeong-sik — COMMENTED** [review]
MASC improver check: the `attention_fields_with_runtime_trust` bridge pattern is the right structural approach — overlay semantics preserve stronger local reasons (pending approvals) while promoting runtime-trust attention when local path reports healthy. Two observations: (1) the "promote trust attention without overriding existing attention reasons" invariant should be documented as a comment on the function itself so future maintainers understand the precedence rule without reading test code. (2) test coverage (41 tests across 3 suites) is thorough — the promotion-vs-preservation boundary cases are well-exercised. This is ready for merge once CI passes.

File: lib/keeper/keeper_status_bridge.ml [hunk 1]
```diff
@@ -411,6 +411,51 @@ let attention_fields_json (config : Coord_utils.config) (meta : keeper_meta) =
     ("next_human_action", Json_util.string_opt_to_json next_human_action);
   ]
 
+let json_string_opt_member key json =
+  match Yojson.Safe.Util.member key json with
+  | `String value ->
+      let trimmed = String.trim value in
+      if trimmed = "" then None else Some trimmed
+  | _ -> None
```
[the remark this answers is not in the document]
Comment (jeong-sik) [reply lib/keeper/keeper_status_bridge.ml]: 동의 — 같은 helper 중복 + argument order 다름. fix:
```ocaml
(* lib/keeper/keeper_status_helpers.ml — single source *)
let json_string_opt_member ~key json =
  match Yojson.Safe.Util.member key json with
  | `String s when String.trim s <> '' -> Some s
  | _ -> None
```
두 file 모두 single helper import. argument order 통일 + extra guarding 보존.

File: lib/keeper/keeper_status_bridge.ml [hunk 2]
```diff
@@ -411,6 +411,51 @@ let attention_fields_json (config : Coord_utils.config) (meta : keeper_meta) =
...
+          | None -> (
+              match json_string_opt_member "latest_next_action" runtime_trust with
+              | Some _ as value -> value
+              | None -> Some "inspect_runtime_trust"))
+    in
+    [
+      ("needs_attention", `Bool true);
+      ("attention_reason", Json_util.string_opt_to_json attention_reason);
+      ("next_human_action", Json_util.string_opt_to_json next_human_action);
+    ]
```
[the remark this answers is not in the document]
Comment (jeong-sik) [reply lib/keeper/keeper_status_bridge.ml]: 동의 — fresh 3-field list가 기존 keys drop. fix:
```ocaml
let promote_runtime_trust ~existing_attention =
  let promoted = [
    "runtime_trust_promoted", `Bool true;
    "trust_score", `Float score;
    "reason", `String reason;
  ] in
  (* Merge: 기존 keys 보존, promoted keys overlay *)
  let merged = List.fold_left (fun acc (k, v) ->
    (k, v) :: List.remove_assoc k acc
  ) existing_attention promoted in
  merged
```
merge semantics — caller가 추가한 keys 안전 보존.
